backend/Events/NightTimeVoteEvents.js

```diff
@@ -17,34 +17,30 @@ exports.loadNightTimeEvents = (io, socket, mafiaGame) => {
      * In future iterations, this will be changed to tally votes from mafia members before setting the chosen player.
      */
     socket.on('mafia-vote', (mafiaVoteObj) => {
-        let room = mafiaGame.gameRoomsDict[socket.player.roomID];
-        room.voteHandler.mafiaVoteMap[socket.player] = room.getPlayerByNickname(mafiaVoteObj.votingFor)
+        const room = mafiaGame.gameRoomsDict[socket.player.roomID];
+        room.voteHandler.mafiaVoteMap[socket.player] = room.getPlayerByNickname(mafiaVoteObj.votingFor);
```

Comment:
could be same issue as #153, use nickname instead of player object

Reply:
yeap fixed as well